tests/test_configuration.py

```diff
@@ -0,0 +1,93 @@
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.test import SimpleTestCase
+
+
+class ConfigurationTestCase(SimpleTestCase):
+    """
+    Test the configuration of settings for the management command.
+    """
```

Comment:
These tests only check user configurations. I tried to use [Moto](https://github.com/spulec/moto) to mock Boto3 for integration tests, but I ran into two main problems:

1. [Not all EC2 endpoints we use are fully implemented](https://github.com/spulec/moto/blob/master/IMPLEMENTATION_COVERAGE.md#ec2---37-implemented), e.g. `describe_subnets`
2. Inscrutable errors showing that mocks were not being applied, as documented in https://github.com/spulec/moto/issues/2069

If 2) above can be overcome, then we could likely support at least partial tests for endpoints that aren't blocked by 1).